messages/en.json

```
@@ -382,5 +413,229 @@
+    "ENV_SETUP_DESC": "Configure your environment variables and API keys",
+    "INSTALLATION": "Installation Guide",
+    "INSTALLATION_DESC": "Step-by-step installation instructions",
+    "USAGE_GUIDE": "Usage Guide",
+    "USAGE_GUIDE_DESC": "Learn how to use and customize your directory",
+    "TECH_STACK": "Technology Stack",
+    "TECH_STACK_DESC": "Explore the technologies powering your directory",
+    "MONETIZATION": "Monetization",
+    "MONETIZATION_DESC": "Learn how to generate revenue from your directory",
+    "SUPPORT": "Support & Help",
+    "SUPPORT_DESC": "Get help and find answers to common questions",
+    "STEP1_FEATURE_1": "Repository cloning",
```

Biome (1.9.4)

[error] The key STEP1_FEATURE_1 was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)